.gitignore

```diff
@@ -86,8 +87,15 @@ contiki-cc2530dk.lib
 *.d71
 *.d81
 
+# Cooja Build Artifacts
+*.cooja
+!Makefile.cooja
```

Comment:
I don't think `!Makefile.cooja` is needed, as the file is already checked in, and no such file is generated (or am I missing something?). Other platforms don't have this line.